lighthouse-cli/test/fixtures/static-server.js

```diff
@@ -60,21 +61,45 @@ function requestHandler(request, response) {
       headers = {'Content-Type': 'image/svg+xml'};
     }
 
-    if (queryString && typeof queryString.status_code !== 'undefined') {
-      statusCode = parseInt(queryString.status_code, 10);
+    if (queryString) {
+      // set document status-code
+      if (typeof queryString.status_code !== 'undefined') {
+        statusCode = parseInt(queryString.status_code, 10);
+      }
+
+      // set delay of request when present
+      if (typeof queryString.delay !== 'undefined') {
+        delay = parseInt(queryString.delay, 10) || 2000;
+      }
+
+      // redirect url to new url if present
+      if (typeof queryString.redirect !== 'undefined') {
+        if (delay > 0) {
+          return setTimeout(sendRedirect, delay, queryString.redirect);
+        } else {
+          return sendRedirect(queryString.redirect);
+        }
+      }
     }
 
     response.writeHead(statusCode, headers);
 
-    // Delay the response by the specified ms defaulting to 2000ms for non-numeric values
-    if (queryString && typeof queryString.delay !== 'undefined') {
-      response.write('');
```

Comment:
was `response.write('');` important here? I have no idea :) but it was here since the first version

Reply:
I don't think so as `response.write('');` as response.end() is important to close the request.
maybe @paulirish still knows why he added it.

Reply:
i barely remember it being important. is it fine if we keep it as is?

Reply:
keep it as is = add the response.write('')?

Reply:
okay scratch what i said. ` response.write('')` is a noop.

`.write('.')` does change things quite considerably, but in a bad direction. (the latency moves from TTFB to content download).

your change here is great. let's keep it. :)